jenkins/Dockerfile-blossom.ubuntu

```diff
@@ -45,7 +45,7 @@ RUN apt-get update -y && \
 RUN add-apt-repository ppa:deadsnakes/ppa && \
     apt-get update -y && \
     DEBIAN_FRONTEND="noninteractive" apt-get install -y maven \
-    openjdk-8-jdk openjdk-11-jdk python3.8 python3.8-distutils python3-setuptools tzdata git
+    openjdk-8-jdk openjdk-11-jdk python3.8 python3.8-distutils python3-setuptools tzdata git zip unzip
```

Comment:
For a separate issue, it looks like Dockerfile-blossom.ubuntu is mostly a superset of Dockerfile-blossom.intergation.ubuntu. So we could either make Dockerfile-blossom.intergation.ubuntu the base image or factor out a significant amount into a common build image to avoid repeatitions.

Reply:
> it looks like Dockerfile-blossom.ubuntu is mostly a superset of Dockerfile-blossom.intergation.ubuntu

Dockerfile-blossom.ubuntu is for premerge/nightly build only,
Dockerfile-blossom.intergation.ubuntu is for integration tests only.

We make them separately for 3 reasons:
1. keep the image as small as possible
2. for common part, yes we could have them in a common image, but for external user who may refer to these dockerfiles, they cannot access our internal docker registry or we must maintain this in a public place like dockerhub.
3. if any issue, we will need to debug in one more extra place, like check if the common image update to date and why its not.

As we do not modify those frequently, I would prefer to keep the way it is for now.